File: app/src/main/java/com/qfq/tainzhi/videoplayer/bean/LocalVideoBean.java
package com.qfq.tainzhi.videoplayer.bean;

/**
 * Created by muqing on 2019/6/11.
 * Email: qfq61@qq.com
 */
public class LocalVideoBean {
    //id, data(path), size, date_added, title, duration,resolution,
    int id;
    String path;
    String size; // long convert to 35M
    String title;
    String duration; // long convert to 100:80:00
    String resolution;
    String date_created;
    String date_modified;
    String date_taken;
    String createDate;
    String thumbnailPath;
    
    public LocalVideoBean(int mId, String mPath, String mSize, String mTitle, String mDuration, String mResolution, String mDate_created, String mDate_modified, String mDate_taken, String mCreateDate, String mThumbnailPath) {
        id = mId;
        path = mPath;
        size = mSize;
        title = mTitle;
        duration = mDuration;
        resolution = mResolution;
        date_created = mDate_created;
        date_modified = mDate_modified;
        date_taken = mDate_taken;
        createDate = mCreateDate;
        thumbnailPath = mThumbnailPath;
    }
    
    public int getId() {
        return id;
    }
    
    public void setId(int mId) {
        id = mId;
    }
    
    public String getPath() {
        return path;
    }
    
    public void setPath(String mPath) {
        path = mPath;
    }
    
    public String getSize() {
        return size;
    }
    
    public void setSize(String mSize) {
        size = mSize;
    }
    
    public String getTitle() {
        return title;
    }
    
    public void setTitle(String mTitle) {
        title = mTitle;
    }
    
    public String getDuration() {
        return duration;
    }
    
    public void setDuration(String mDuration) {
        duration = mDuration;
    }
    
    public String getResolution() {
        return resolution;
    }
    
    public void setResolution(String mResolution) {
        resolution = mResolution;
    }
    
    public String getDate_created() {
        return date_created;
    }
    
    public void setDate_created(String mDate_created) {
        date_created = mDate_created;
    }
    
    public String getDate_modified() {
        return date_modified;
    }
    
    public void setDate_modified(String mDate_modified) {
        date_modified = mDate_modified;
    }
    
    public String getDate_taken() {
        return date_taken;
    }
    
    public void setDate_taken(String mDate_taken) {
        date_taken = mDate_taken;
    }
    
    public String getCreateDate() {
        return createDate;
    }
    
    public void setCreateDate(String mCreateDate) {
        createDate = mCreateDate;
    }
    
    public String getThumbnailPath() {
        return thumbnailPath;
    }
    
    public void setThumbnailPath(String mThumbnailPath) {
        thumbnailPath = mThumbnailPath;
    }
    
    @Override
    public String toString() {
        return "LocalVideoBean{" +
                       "id=" + id +
                       ", path='" + path + '\'' +
                       ", size='" + size + '\'' +
                       ", title='" + title + '\'' +
                       ", duration='" + duration + '\'' +
                       ", resolution='" + resolution + '\'' +
                       ", date_created='" + date_created + '\'' +
                       ", date_modified='" + date_modified + '\'' +
                       ", date_taken='" + date_taken + '\'' +
                       ", createDate='" + createDate + '\'' +
                       ", thumbnailPath='" + thumbnailPath + '\'' +
                       '}';
    }
}

File: app/src/main/java/com/qfq/tainzhi/videoplayer/bean/LocalVideoBean.kt
package com.qfq.tainzhi.videoplayer.bean

/**
 * Created by muqing on 2019/6/11.
 * Email: qfq61@qq.com
 */
class LocalVideoBean constructor(
        //id, data(path), size, date_added, title, duration,resolution,
        var id: Int, var path: String?, // long convert to 35M
        var size: String?, var title: String?,
        //多少秒
        var duration: Long, var resolution: String?,
        var date_created: String?, var date_modified: String?, var date_taken: String?, var createDate: String?, var thumbnailPath: String?) {
    fun getId(): Int {
        return id
    }
    
    fun setId(mId: Int) {
        id = mId
    }
    
    fun getPath(): String? {
        return path
    }
    
    fun setPath(mPath: String?) {
        path = mPath
    }
    
    fun getSize(): String? {
        return size
    }
    
    fun setSize(mSize: String?) {
        size = mSize
    }
    
    fun getTitle(): String? {
        return title
    }
    
    fun setTitle(mTitle: String?) {
        title = mTitle
    }
    
    fun getDuration(): Long {
        return duration
    }
    
    fun setDuration(mDuration: Long) {
        duration = mDuration
    }
    
    fun getResolution(): String? {
        return resolution
    }
    
    fun setResolution(mResolution: String?) {
        resolution = mResolution
    }
    
    fun getDate_created(): String? {
        return date_created
    }
    
    fun setDate_created(mDate_created: String?) {
        date_created = mDate_created
    }
    
    fun getDate_modified(): String? {
        return date_modified
    }
    
    fun setDate_modified(mDate_modified: String?) {
        date_modified = mDate_modified
    }
    
    fun getDate_taken(): String? {
        return date_taken
    }
    
    fun setDate_taken(mDate_taken: String?) {
        date_taken = mDate_taken
    }
    
    fun getCreateDate(): String? {
        return createDate
    }
    
    fun setCreateDate(mCreateDate: String?) {
        createDate = mCreateDate
    }
    
    fun getThumbnailPath(): String? {
        return thumbnailPath
    }
    
    fun setThumbnailPath(mThumbnailPath: String?) {
        thumbnailPath = mThumbnailPath
    }
    
    public override fun toString(): String {
        return ("LocalVideoBean{" +
                "id=" + id +
                ", path='" + path + '\'' +
                ", size='" + size + '\'' +
                ", title='" + title + '\'' +
                ", duration='" + duration + '\'' +
                ", resolution='" + resolution + '\'' +
                ", date_created='" + date_created + '\'' +
                ", date_modified='" + date_modified + '\'' +
                ", date_taken='" + date_taken + '\'' +
                ", createDate='" + createDate + '\'' +
                ", thumbnailPath='" + thumbnailPath + '\'' +
                '}')
    }
    
}